src/templates/layouts/pressRelease/index.test.tsx

```diff
@@ -109,13 +141,64 @@ describe('<pressRelease> with valid data', () => {
       expect(
         screen.getByText(`${contact.name}, ${contact.description}`)
       ).toBeInTheDocument()
-      expect(screen.getByText(contact.phone)).toBeInTheDocument()
       const emailLink = screen.getByTestId('press-email')
       expect(emailLink).toBeInTheDocument()
       expect(emailLink).toHaveAttribute('href', `mailto:${contact.email}`)
+      expect(screen.getByTestId('phone-0')).toBeInTheDocument()
```

Comment:
This could pass if it's an empty `<div data-testId="phone-0 />`. We should expect this to have some specific content, not just to exist.